lessons/6_objects_intro/index.html

```diff
@@ -0,0 +1,332 @@
+            </div>
+        </section>
+
+        <section>
+            <h3>Objects in a nutshell</h3>
+            <ul>
+                <li class="fragment">Objects consist of keys and values</li>
+                <li class="fragment">The key is like the name of the variable</li>
+                <li class="fragment">The value is like a variable itself</li>
+            </ul>
+            <br/>
+            <div class="fragment">
+                            <pre><code>
+          let book = {
+            title: "Zombie Raccoons & Killer Bunnies",
+            image: "zombie_raccoons.jpg",
+            pageCount: 643,
+            price: 20
+          };
+        </code></pre>
+            </div>
+        </section>
+        <section>
+            <h2>What can we do with objects?</h2>
+        </section>
+        <section>
+            <h3>Get value of it's attributes</h3>
+            <pre><code data-trim>
+              let person = {
+                name: "Anna",
+                age: 29,
+              };
+
+              alert(person.name + " is " + person.age + " years old");
+            </code></pre>
+        </section>
+        <section>
+            <h3>Change value of it's attributes</h3>
+            <pre><code data-trim>
+              let person = {
+                name: "Anna",
+                age: 29,
+              };
+
+              person.age = 17; // Happy birthday!
+
+              alert(person.name + " is " + person.age + " again");
+            </code></pre>
+        </section>
+
+        <section>
+            <h3>And even add new attributes</h3>
+            <pre><code data-trim>
+              let person = {
+                name: "Anna",
+                age: 17,
+              };
+              person.hobby = "just chilling";
+
+              alert(person.name + "'s hobby is " + person.hobby);
+            </code></pre>
+        </section>
+
+        <section>
+            <h3>Objects in real life</h3>
+            <p class="fragment">We've seen them before: Every time we saw a dot! Do you remember?</p>
+            <div class="fragment">
+                <div>
+                    <button id="strangeButton" style="font-size: 20px">That strange button</button>
+                </div>
+                <br/>
+                <div>
+                    <pre><code>
+let strangeButtonCmp = document.getElementById("strangeButton")
+strangeButtonCmp.classList.add('new-class');
+console.log(strangeButtonCmp.classList);
+            </code></pre>
+                </div>
+
+            </div>
+            <p class="fragment">Values inside objects can be anything, even functions or objects!</p>
+        </section>
+
+        <section>
+            <h3>Nested Objects</h3>
+            <p class="fragment">Objects inside objects are called "nested objects":</p>
+            <div class="fragment">
+                            <pre><code data-trim>
+              let person = {
+                name: "Anna",
```

Comment:
This example is copy-pastable, because it's a classwork :)

Reply:
Well, yes, I thought it is ok when classwork is somewhat copy-pastable. But students would have to add 2 more attributes to this object, add another similar objects, and also extract favorite books into separate object. So I guess it has enough complexity for classwork..